code/src/components/AddTodo.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import todos from "../reducers/todos";
+
+const InputContainer = styled.div`
+  display: flex;
+  padding: 20px;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (min-width: 768px) {
+    margin-top: 20px;
+    margin-left: 80px;
+    margin-right: 80px;
+    margin-bottom: 20px;
+  }
+  @media (min-width: 1366px) {
+    justify-content: center;
+  }
+`;
+
+const StyledInput = styled.input`
+  width: 65%;
+  background-color: #8fbfd7;
+  border: 1px solid #337b9e;
+  border-radius: 20px;
+  padding: 10px;
+
+  @media (min-width: 768px) {
+    border-radius: 30px;
+    padding: 25px;
+  }
+  @media (min-width: 1366px) {
+    width: 400px;
+    margin-right: 30px;
+  }
+`;
+
+const AddButton = styled.button`
+  height: 25px;
+  width: 50px;
+  background-color: #3b8cb4;
+  border: 1px solid #337b9e;
+  border-radius: 20px;
+
+  @media (min-width: 768px) {
+    width: 80px;
+    border-radius: 40px;
+
+    &:hover {
+      background-color: #276a8b;
+    }
+  }
+`;
+
+const AddEmoji = styled.span``;
+
+const AddTodo = () => {
+  const [input, setInput] = useState("Add new task ...");
+  const dispatch = useDispatch();
+  const onAddTodo = (input) => {
+    dispatch(todos.actions.addTodo(input));
+
+    setInput("Add new task ...");
```

Comment:
Do you think that it would be better to have the "Add new task ..." as a placeholder in the text input in order not to have to erase it manually every time that we want to add a new task?

Reply:
Yes, thank you!! I wanted to add that but did not have time to figure it out :)